docs/contributing.md

```diff
@@ -0,0 +1,143 @@
+# Contributors Guide
+
+This is a community driven project and everyone is welcome to contribute. The
+project is hosted at the [gmt-examples GitHub repository](https://github.com/GenericMappingTools/gmt-examples).
+
+The goal is to maintain a diverse community that's pleasant for everyone.
+**Please be considerate and respectful of others**. Everyone must abide by our
+[Code of Conduct](https://github.com/GenericMappingTools/gmt-examples/blob/main/CODE_OF_CONDUCT.md)
+and we encourage all to read it carefully.
+
+## Setting up your environment
+
+The following dependencies are required for building the GMT examples pages:
+
+- [gmt](https://docs.generic-mapping-tools.org/latest/)
+- [myst-parser](https://myst-parser.readthedocs.io/en/latest/)
+- [sphinx](https://www.sphinx-doc.org/en/master/)
+- [sphinx_gmt](https://www.generic-mapping-tools.org/sphinx_gmt/latest/)
+- [sphinx_rtd_theme](https://sphinx-rtd-theme.readthedocs.io/en/stable/installing.html)
+
+Two options for installing the dependencies are [pip](#pip-setup) and [conda](#conda-setup).
```

Comment:
Might be worth mentioning here that sphinx and the other things needed to build the website are Python based and this is why Python is needed (and pip and conda are options).

Reply:
Also mentioning that they'll need a working Python environment, however they want to set one up. Then we can say that if they don't know how to do that, installing [Miniforge](https://github.com/conda-forge/miniforge) and using the `conda` environment below is the easiest way to get everything setup. If they know what they're doing, then they can do whatever and avoid conda if desired.

I recommend miniforge here instead of Anaconda because it really only comes with the bare minimum and is (somewhat) independent from Anaconda Inc.